package.json

```diff
@@ -89,18 +89,11 @@
   "devDependencies": {
     "auth0-extensions-cli": "^4.0.4",
     "chai": "^4.1.0",
-    "eslint": "^4.3.0",
-    "eslint-config-auth0": "^11.0.0",
-    "eslint-config-auth0-base": "^13.0.0",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-react": "^7.1.0",
```

Comment:
Any context on the removal of `eslint`?

Reply:
without removing npm ci was not working on any of the node versions [here](https://auth0.slack.com/archives/C03RFK866V7/p1728664728343839)